integration-tests/docker/environment-configs/common

```diff
@@ -39,6 +39,7 @@ druid_auth_authenticatorChain=["basic"]
 druid_auth_authorizer_basic_type=basic
 druid_auth_authorizers=["basic"]
 druid_auth_authorizeQueryContextParams=true
+druid_policy_enforcer_type=none
```

Comment:
Let's not include this here, same reason as we shouldn't include in the default configs.